pkg/config/config.go

```diff
@@ -446,6 +446,7 @@ func initConfig(config Config) {
 	config.SetKnown("system_probe_config.max_closed_connections_buffered")
 	config.SetKnown("system_probe_config.max_connection_state_buffered")
 	config.SetKnown("system_probe_config.excluded_linux_versions")
+	config.SetKnown("system_probe_config.skip_linux_version_check")
```

Comment:
Will this setup a ENV flag for this variable as well? Would be helpful to make sure one bound to this flag

Reply:
I think a word is missing from your comment, do we want an ENV flag for this or we don't ?